index.html

```diff
@@ -0,0 +1,85 @@
+<!-- Danielle Kunkel pr2 card design -->
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link
+        href="https://fonts.googleapis.com/css2?family=Libre+Baskerville:wght@400;700&family=Montserrat:wght@700&display=swap"
+        rel="stylesheet">
+    <title>How 'bioliphic' design can create a better workspace</title>
+</head>
+
+<body>
+
+    <article>
+        <a href="https://www.bbc.com/worklife/article/20200929-how-biophilic-design-can-create-a-better-workspace"
+            target="blank" <div class="image-wrapper">
+            <img src="https://cdn.glitch.com/5e9189cf-962e-4e13-a403-b7e677750267%2FcroppedTerracotta.jpg?v=1603939526784"
+                alt="green plants in terracotta pots on white shelves" />
+        </a>
+        </div>
```

Comment:
This HTML got a bit broken:
1. The `a` element on line 18 is missing the final `>` bracket.
1. On line 18, the `a` element is before the `div`, but on lines 22-23, that is reversed.

```suggestion
        <div class="image-wrapper">
            <a href="https://www.bbc.com/worklife/article/20200929-how-biophilic-design-can-create-a-better-workspace" target="blank">
                <img src="https://cdn.glitch.com/5e9189cf-962e-4e13-a403-b7e677750267%2FcroppedTerracotta.jpg?v=1603939526784" alt="green plants in terracotta pots on white shelves" />
            </a>
        </div>
```